scripts/imagebuild.sh

```diff
@@ -23,9 +23,15 @@ dockerfile_dir="${base_dir}/images/federation-v2"
 
 [ -f "$base_dir/bin/controller-manager" ] || { echo "$base_dir/bin/controller-manager not found" ; exit 1 ;}
 
-if [[ "${TRAVIS_BRANCH}" != "master" ]]; then
-  echo "Not on master branch. Image build skipped." >&2
-  exit 0
+if [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]?\d$ ]]; then
+    echo "Pushing images with tags '${TRAVIS_TAG}' and 'latest'."
```

Comment:
What do you mean of `tags '${TRAVIS_TAG}' and 'latest'.`? Why including `latest` here?

Reply:
we are not pushing the image with latest tag. either message needs updating or we should push image with latest tag.